completions/help

```diff
@@ -6,7 +6,7 @@ _comp_cmd_help()
     _comp_initialize -- "$@" || return
 
     if [[ $cur == -* ]]; then
-        COMPREPLY=($(compgen -W '$(_parse_usage "$1" "-s $1")' -- "$cur"))
+        _comp_compgen_usage -c help -s "$1"
```

Comment:
Maybe
```suggestion
        _comp_compgen_usage -- -s "$1"
```

Reply:
I wanted to make it consistent with the other builtins. It is technically the last word `"$1"` that specifies the name of the current command but not `comp_args[0]` internally recorded.

```bash
./completions/alias:22:        _comp_compgen_usage -c help -s "$1"
./completions/bind:28:        _comp_compgen_usage -c help -s "$1"
./completions/cd:11:        _comp_compgen_help -c help "$1"
./completions/help:9:        _comp_compgen_usage -c help -s "$1"
./completions/kill:21:        _comp_compgen -a help -c help "$1"
./completions/pwd:14:    _comp_compgen_usage -c help -s "$1"
./completions/ulimit:30:            _comp_compgen_usage -c help -s "$1"
```